src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
```

Comment:
![image](https://github.com/AbuTalha3/bookstore-app/assets/93039359/a55275a8-ebea-4ddb-86bc-d402784078a9)

- [ ] It would be nice if you fix the highlighted link in the image above 👀 because it is broken 😨 Kindly fix it or remove it **(If you don't have any dedicated section for this link)** to make your readme descriptive and professional 👍